src/olympia/accounts/tests/test_utils.py

```diff
@@ -247,20 +268,39 @@ def test_success(self, primary_email_change_event):
         process_fxa_event(self.body)
         primary_email_change_event.assert_called()
         primary_email_change_event.assert_called_with(
-            'new-email@example.com', 'ABCDEF012345689',
+            'ABCDEF012345689',
+            totimestamp(self.email_changed_date),
+            'new-email@example.com')
+
+
+class TestProcessFxAEventDelete(TestCase):
+    def setUp(self):
+        self.email_changed_date = self.days_ago(42)
+        self.body = json.dumps({'Message': json.dumps(
+            {'event': 'delete',
+             'uid': 'ABCDEF012345689',
+             'ts': totimestamp(self.email_changed_date)})})
+
+    @override_switch('fxa-account-delete', active=True)
+    def test_success_integration(self):
+        user = user_factory(fxa_id='ABCDEF012345689')
```

Comment:
How about setting this into `self.uid` in `setUp` and then referencing that variable here, rather than relying on accurate hardcoding?

Reply:
https://github.com/mozilla/addons-server/pull/14546/commits/b4ec801141c93ca71daf6459ed7b73118d9182a5